src/mac.cpp

```diff
@@ -304,6 +304,30 @@ std::string GetActiveApplication() {
   return [[NSString stringWithFormat:@"%@", running.bundleURL.path] UTF8String];
 }
 
+bool ActiveApplicationIsSandboxed() {
+  int pid = GetActivePid();
+  NSRunningApplication* running = [NSRunningApplication runningApplicationWithProcessIdentifier:pid];
+  NSURL *bundleURL = running.bundleURL;
+  SecStaticCodeRef staticCode = NULL;
+  bool isSandboxed = false;
+
+  if (SecStaticCodeCreateWithPath((__bridge CFURLRef)bundleURL, kSecCSDefaultFlags, &staticCode) == errSecSuccess) {
+    if (SecStaticCodeCheckValidityWithErrors(staticCode, kSecCSBasicValidateOnly, NULL, NULL) == errSecSuccess) {
+      SecRequirementRef sandboxRequirement;
+      if (SecRequirementCreateWithString(CFSTR("entitlement[\"com.apple.security.app-sandbox\"] exists"), kSecCSDefaultFlags,
```

Comment:
Does this require a CFRelease? I noticed "Create" in the name.

Reply:
It does. Added in 27fc0d7